src/main/java/org/fogbowcloud/saps/engine/core/catalog/Catalog.java

```diff
@@ -0,0 +1,52 @@
+package org.fogbowcloud.saps.engine.core.catalog;
+
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.List;
+
+import org.fogbowcloud.saps.engine.core.catalog.exceptions.TaskNotFoundException;
+import org.fogbowcloud.saps.engine.core.catalog.exceptions.UserNotFoundException;
+import org.fogbowcloud.saps.engine.core.dispatcher.notifier.Ward;
+import org.fogbowcloud.saps.engine.core.model.SapsImage;
+import org.fogbowcloud.saps.engine.core.model.SapsUser;
+import org.fogbowcloud.saps.engine.core.model.enums.ImageTaskState;
+
+public interface Catalog {
+
+    int UNLIMITED = -1;
```

Comment:
Remove this field from the interface. Create a class to store catalog constants.

Reply:
Done